virtdeploy/cli.py

```diff
@@ -40,7 +41,8 @@
 
 def instance_create(args):
     driver = virtdeploy.get_driver(DRIVER)
-    instance = driver.instance_create(args.id, args.template)
+    args.settings = yaml.load(open(args.settings, 'r')) if (args.settings is not None) else {}
```

Comment:
Don't override variables. This would do:

    settings = yaml.load(open(args.settings, 'r')) if args.settings is not None else {}